docs/components/InteractiveWell.tsx

```diff
@@ -0,0 +1,54 @@
+import { NetworkSwitcher } from "./atoms/NetworkSwitcher";
+
+export const InteractiveWell = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
+  return (
+    <section className="px-3 pb-3 my-3 rounded">
+      <div className="badge text-xs font-weight-bold">Live Minting Demo</div>
```

Comment:
We want this to be a reusable component so this shouldn't say `Live Minting Demo`, instead it should have a `{title}` param. 

Let's also not hard code the margin but allow this to accept a `className` prop.